pootle/static/js/admin/duedates/components/DueDatePicker.css

```diff
@@ -165,12 +265,6 @@
 
 /* Modifiers */
```

Comment:
Do you mean that you would prefer to keep the stock styles intact and reset/override them later? Or just move a few new CSS blocks down to the Modifiers section? I have a feeling that any style resetting and overriding will continue to be tightly coupled with the stock styles, so if something changes radically in the way the component is rendered, our overrides will be broken, anyway. I'm more inclined with dealing with style upgrades when/if we need to switch to a newer version of the date picker component, and keep the styles more predictable and logically organized by altering the style set directly.